apps/ai_reports/serializers.py

```diff
@@ -7,7 +7,9 @@ class AiReportSerializer(serializers.ModelSerializer):
     class Meta:
         model = AiReport
         fields = (
-            "explanation",
             "category",
+            "confidence",
+            "explanation",
+            "is_pending",
```

Comment:
@philli-m I added here the rest of the model's field in case we need to show them on the moderation dashboard. Though in the design looks like only the explanation is displayed: https://app.zeplin.io/project/6492a63d7e162b416cfa77a0/screen/6492a6a3e106304106746c00.
What do u think?